membership-common/src/main/scala/com/gu/memsub/subsv2/Plan.scala

```diff
@@ -229,43 +227,35 @@ object FrontendId {
 
 object CatalogPlan {
 
-  type Member = CatalogPlan[Product.Membership, ChargeList with SingleBenefit[MemberTier], Current]
+  type Supporter[+B <: BillingPeriod] = CatalogPlan[Product.Membership, SingleCharge[Supporter.type, B], Current]
+  type Partner[+B <: BillingPeriod] = CatalogPlan[Product.Membership, SingleCharge[Partner.type, B], Current]
+  type Patron[+B <: BillingPeriod] = CatalogPlan[Product.Membership, SingleCharge[Patron.type, B], Current]
 
-  type PaidMember[+B <: BillingPeriod] = CatalogPlan[Product.Membership, PaidCharge[PaidMemberTier, B], Current]
```

Comment:
Member and PaidMember were unused and deleted, not sure why the diff is so messed up, the rest just changed from PaidCharge to SingleCharge